bpf/flow.h

```diff
@@ -5,11 +5,23 @@
 #define TC_ACT_SHOT 2
 #define IP_MAX_LEN 16
 
+// TODO : Explore if this can be programmed from go launcher
+// Total mem consumption would be (1000*)
+#define INGRESS_MAX_ENTRIES 1000
+#define EGRESS_MAX_ENTRIES  1000
```

Comment:
Yes, this can be configured from the Go launcher. As an example, the existing sampling configuration value:

1. [Define volatile const variables](https://github.com/netobserv/netobserv-ebpf-agent/blob/main/bpf/flows.c#L30)
2. [Use the RewriteConstants method from Go](https://github.com/netobserv/netobserv-ebpf-agent/blob/main/pkg/ebpf/tracer.go#L60-L62)

Reply:
@mariomac : When max entries are reached, newer flows are sent to userspace for aggregation via ringbuffer (as in the current method).
Have more future optimizations in this area if we observe more activity in userspace, like  Eviction can be made more aggressive. Studying Real-world ocp traces would give us an idea if optimizations are needed in that directions.

Thanks for sharing the rewriteconstants, I will check it out.

Reply:
Tried this, apparently using volatile const for map size is not supported. It fails during compilation.
 error: fields must have a constant size: 'variable length array in structure' extension will never be supported
    __uint(max_entries, ingress_max_entries);